CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # CHANGELOG
 
+## `v19.0.0`
+
+### Breaking Changes
+
+|            Package Name |      API Version      |
+| ----------------------: | :-------------------: |
+|               authoring |          v2.0         |
+|              automation | 2015-10-31<br/>2017-05-15-preview<br/>2018-01-preview |
+|                  backup | 2016-12-01<br/>2017-07-01 |
+|             botservices |       2017-12-01      |
+|                 compute | 2015-06-15<br/>2016-03-30<br/>2017-03-30<br/>2017-12-01<br/>2018-04-01<br/>2016-04-30-preview |
+|          computervision |          v2.0         |
+|       containerregistry |       2017-10-01      |
+|               eventgrid |       2018-01-01      |
+|               hdinsight | 2018-06-01-preview |
+|                insights | 2017-05-01-preview<br/>2018-03-01 |
+|                 migrate |       2018-02-02      |
+|                 runtime |          v2.0         |
+|                     sql |   2017-03-01-preview  |
+|                     web |       2018-02-01      |
+
```

Comment:
Qq, how did you obtain this ? for brekaing changes usually I had a small JS script to theck which ones have the breaking changes field or not. is there a beter way to do it now ?

Reply:
I run the apidiff tool then manually parse the results to build the tables.  I'd like to build a tool that takes the output from apidiff and spits out the table(s) but I just haven't had time to do it.